pkg/agent/ipassigner/ip_assigner_linux.go

```diff
@@ -31,25 +32,188 @@ import (
 	"antrea.io/antrea/pkg/agent/util/arping"
 	"antrea.io/antrea/pkg/agent/util/ndp"
 	"antrea.io/antrea/pkg/agent/util/sysctl"
+	crdv1b1 "antrea.io/antrea/pkg/apis/crd/v1beta1"
 )
 
-// ipAssigner creates a dummy device and assigns IPs to it.
+// VLAN interfaces created by antrea-agent will be named with the prefix.
+// For example, when vlan ID is 10, the name will be antrea-ext.10.
+// It can be used to determine whether it's safe to delete an interface when it's no longer used.
+const vlanInterfacePrefix = "antrea-ext."
```

Comment:
Do you feel better to keep the VLAN interface naming convention like eth0.10, ens192.100, etc? Maybe you need a hardcoded prefix to know if an interface is created by Antrea or not?

Reply:
I don't know which one is better but this is a trade-off we have to choose. I chose to have a antrea specific prefix to make it clear it's created by antrea so we can remove it safely, and user may easily get it. We can name it using eth0.10, but we will need another way like a file storage to know which are managed by us, otherwise it would end up with removing all VLAN interfaces users created for other purposes.

Reply:
Sure. Agreed the "antrea-" prefix is the simplest way to identify the interfaces.